load to timescale fails with  pq: invalid input syntax for type json

I have an AWS instance with Ubuntu  16.04.5 LTS where I've installed go 1.9 and postgres 9.6.10 (from http://apt.postgresql.org/pub/repos/apt/ ) and the current timescale (using https://blog.timescale.com/tutorial-installing-timescaledb-on-aws-c8602b767a98). I then installed tsbs and generated a timescale database with 
bin/tsbs_generate_data -use-case="devops" -seed=123 -scale-var=8000 -timestamp-start="2018-08-08T00:00:00Z" -timestamp-end="2018-08-09T00:00:00Z" -log-interval="5s" -format="timescaledb" | gzip > tsbs-devops-8000-1day-5s-timescaledb-data.gz
 Attempting to load I used load_timescaledb.sh with these environment variables set
postgres@ip-172-31-18-107:~$ export PATH="/home/ubuntu/gosrc/bin:$PATH"
postgres@ip-172-31-18-107:~$ export GOPATH=/home/ubuntu/gosrc
postgres@ip-172-31-18-107:/data$ export DATA_FILE=/data/tsbs-devops-8000-1day-5s-timescaledb-data.gz

postgres@ip-172-31-18-107:/data$ /home/ubuntu/gosrc/src/github.com/timescale/tsbs/scripts/load_timescaledb.sh
Bulk loading file /data/tsbs-devops-8000-1day-5s-timescaledb-data.gz
+ source /home/ubuntu/gosrc/src/github.com/timescale/tsbs/scripts/timescaledb.conf
++ JSON_TAGS=false
++ DATABASE_NAME=benchmark
++ IN_TABLE_PARTITION_TAG=true
++ USE_HYPERTABLE=true
+ pg_isready -h localhost
localhost:5432 - accepting connections
+ cat /data/tsbs-devops-8000-1day-5s-timescaledb-data.gz
+ gunzip
+ tsbs_load_timescaledb --postgres=sslmode=disable --db-name=benchmark --host=localhost --user=postgres --workers=8 --batch-size=10000 --reporting-period=10s --use-hypertable=true --use-jsonb-tags=false --in-table-partition-tag=true --hash-workers=false --time-partition-index=false --partitions=1 --chunk-time=8h --write-profile= --field-index-count=1
time,per. metric/s,metric total,overall metric/s,per. row/s,row total,overall row/s
panic: pq: invalid input syntax for type json

goroutine 51 [running]:
main.(*processor).processCSI(0xc420622000, 0xc42348fe00, 0x5, 0xc423bed000, 0x7d0, 0x900, 0xc4204e3400)
	/home/ubuntu/gosrc/src/github.com/timescale/tsbs/cmd/tsbs_load_timescaledb/process.go:186 +0x1888
main.(*processor).ProcessBatch(0xc420622000, 0x7df9c0, 0xc423a1df50, 0x1, 0x0, 0x0)
	/home/ubuntu/gosrc/src/github.com/timescale/tsbs/cmd/tsbs_load_timescaledb/process.go:232 +0x192
github.com/timescale/tsbs/load.(*BenchmarkRunner).work(0x7fc000, 0x7e2020, 0x81afc0, 0xc420528a10, 0xc42052d780, 0x7)
	/home/ubuntu/gosrc/src/github.com/timescale/tsbs/load/loader.go:202 +0xdd
created by github.com/timescale/tsbs/load.(*BenchmarkRunner).RunBenchmark
	/home/ubuntu/gosrc/src/github.com/timescale/tsbs/load/loader.go:105 +0x156


I am able to load to influx.  What versions should I be using?

--- a/cmd/tsbs_load_timescaledb/process.go
+++ b/cmd/tsbs_load_timescaledb/process.go
@@ -31,7 +31,7 @@ func newSyncCSI() *syncCSI {
 var globalSyncCSI = newSyncCSI()
 
 func subsystemTagsToJSON(tags []string) string {
-	json := "'{"
+	json := "{"
 	for i, t := range tags {
 		args := strings.Split(t, "=")
 		if i > 0 {
@@ -39,7 +39,7 @@ func subsystemTagsToJSON(tags []string) string {
 		}
 		json += fmt.Sprintf("\"%s\": \"%s\"", args[0], args[1])
 	}
-	json += "}'"
+	json += "}"
 	return json
 }
 

